terraform/environments/core-logging/monitoring.tf

```diff
@@ -2,3 +2,29 @@ module "core_monitoring" {
   source                     = "../../modules/core-monitoring"
   pagerduty_integration_keys = local.pagerduty_integration_keys
 }
+resource "aws_cloudwatch_log_group" "aws_route53_logs_com" {
```

Comment:
Can you update the resource name here to something like `"route53-resolver-logs"`?

Reply:
Amended to aws_route53_logs I think